packages/app/src/app/pages/NewSandbox/index.js

```diff
@@ -16,8 +16,14 @@ class NewSandboxComponent extends React.PureComponent {
     this.props.signals.sandboxPageMounted();
   }
 
-  createSandbox = template => {
-    history.push(sandboxUrl({ id: template.shortid }));
+  createSandbox = (template, e) => {
```

Comment:
I think it'd be nicer to have the second argument be something like `options: { openInNewTab: boolean }`.